actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/UserBulkMigrationActor.java

```diff
@@ -408,8 +408,8 @@ private String getCreatedBy(Request request) {
             case "school udise id":
               mappedColumns.add("schoolUdiseId");
               break;
-            case "diksha sub-org id":
-              mappedColumns.add("subOrgId");
+            case "sub org ext id":
```

Comment:
Aren't we supposed to remove dependency on this column? 
The school udise code is nothing but org external id.

Reply:
From latter discussion org external id is taken from school udise code.